autosklearn/automl.py

```diff
@@ -210,7 +210,7 @@ def __init__(
         get_smac_object_callback: Optional[Callable] = None,
         smac_scenario_args: Optional[Mapping] = None,
         logging_config: Optional[Mapping] = None,
-        metric: Optional[Scorer] = None,
+        metric: Optional[Scorer | Sequence[Scorer]] = None,
```

Comment:
Not neccessary, just something to know `Optional[X] == Union[X, None] == X | None`
i..e you could write `Scorer | Sequence[Scorer] | None = None`